index.js

```diff
@@ -29,58 +29,92 @@ const availableCommandsMessage = `Available commands are:-
  */
 module.exports = app => {
 	/* --------------------------------- ANCHOR Automation --------------------------------- */
-	app.on('pull_request.opened', addReadyForReviewLabel);
+	app.on("pull_request.opened", addReadyForReviewLabel);
 
-	app.on('pull_request_review.submitted', addApprovedLabel);
-	app.on('pull_request_review.submitted', changesRequestLabel);
+	app.on("pull_request_review.submitted", addApprovedLabel);
+	app.on("pull_request_review.submitted", changesRequestLabel);
 
-	app.on('pull_request.closed', addMergedLabel);
+	app.on("pull_request.closed", addMergedLabel);
+	app.on("pull_request.closed", async context => {
```

Comment:
Codacy has a fix for the issue: [Replace `↹app.on("pull_request.closed",·async·context` with `··app.on("pull_request.closed",·async·(context)`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060881)
```suggestion
  app.on("pull_request.closed", async (context) => {
```